.dockerignore

```diff
@@ -1,8 +0,0 @@
-apps/**/node_modules
-apps/catalogue3
-apps/graphql-playground
-apps/pages
-apps/schema
-apps/settings
-apps/tables
-apps/updownload
```

Comment:
why is this removed ? if i remember correct this is done to have a small ssr-catalogue container , only the nuxt stuff is included

Reply:
if a new docker container is needed for the java build we should check for conflict

Reply:
yes indeed. You may notice that the way on how docker images are created has been refactored?